src/replicache.ts

```diff
@@ -650,10 +738,10 @@ export default class Replicache implements ReadTransaction {
       let pullAuth;
       try {
         // Don't want to say we are syncing when we are waiting for auth
-        this._changeSyncCounter(-1);
+        this._changeSyncCounters(0, -1);
```

Comment:
I think it makes sense to say we are syncing when we're waiting for auth.

Reply:
The purpose of `onSync` is to tell users there is some network activity happening related to getting new data. Auth counts as part of that.

Reply:
getPullAuth is not necessary network but could be UI. I would be fine changing this.

Reply:
I see. I think in the case an app is displaying UI they will want to turn off the spinner. But in the case they are just exchanging a refresh token for a new short-term token or something they would *not* want to turn the spinner off (it would look weird to see it glitch off, then back on a second later). So perhaps we need a different API if we want to support UI-based things. It seems like it would make it easier to remove the feature that we'll "turn off sync progress" during auth check, so I support doing that, until someone asks for it.